### DISCUSSION TITLE
Important Question

### DISCUSSION THREAD

**Discussion by twist0R** [twist0R/tw_policebadge#1] [Q&A] [answered]
What do you think about me doing a big Update and putting this script to work for QBCore, NPFramework and even vRP

**Comment by globodesk32** [comment, marked as answer]
can you help me with this script? with my mdt doesn't works

**Reply by twist0R** [reply]
SELECT * FROM user_mdt INNER JOIN users ON user_mdt.char_id=users.id where users.identifier = @identifier

**Comment by globodesk32** [comment]
i will test if works thanks :D
